addons/website_event_track/views/event_track_templates_page.xml

```diff
@@ -99,37 +117,39 @@
-                            <i class="fa fa-briefcase mr-2"/><span t-out="track.partner_function"/>
-                            <t t-if="track.partner_company_name">
-                                <span>&amp;nbsp;at&amp;nbsp;</span><span t-out="track.partner_company_name"/>
-                            </t>
-                        </span>
-                        <span class="mb-1 d-flex align-items-baseline text-break" t-if="track.partner_id.website">
-                            <i class="fa fa-home mr-2"/><a t-att-href="track.partner_id.website"><span t-field="track.partner_id.website"/></a>
-                        </span>
-                        <span class="mb-1 d-flex align-items-baseline text-break" t-if="track.partner_email">
-                            <i class="fa fa-envelope mr-2"/><a t-att-mailto="track.partner_email"><span t-field="track.partner_email"/></a>
-                        </span>
-                        <span class="mb-1 d-flex align-items-baseline text-break" t-if="track.partner_phone">
-                            <i class="fa fa-phone mr-2"/><span t-field="track.partner_phone"/>
-                        </span>
+            <t t-if="track.image or track.partner_name or track.partner_function or track.partner_id.website or track.partner_email or track.partner_phone or not is_html_empty(track.partner_biography)">
+                <hr class="mt-2 mb-0"/>
+                <div class="mx-3">
+                    <div class="mt-2 mb-2 d-flex">
+                        <div t-if="track.image" t-field="track.image"
+                             t-options="{'widget': 'image', 'class': 'rounded-circle mr-2', 'max_width': '96'}"/>
+                        <div class="pl-2 pr-0 pr-md-2 d-flex flex-grow-1 flex-column">
+                            <div t-if="track.partner_name" t-field="track.partner_name" class="font-weight-bold mb-2"/>
+                            <div class="d-none d-lg-block">
+                                <t t-call="website_event_track.event_track_content_partner_info"/>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="d-block d-lg-none mt-3 mb-3">
+                        <t t-call="website_event_track.event_track_content_partner_info"/>
                     </div>
+                    <div t-if="not is_html_empty(track.partner_biography)" t-field="track.partner_biography"
+                         class="oe_no_empty"/>
+                    <hr t-if="not is_html_empty(track.description)" class="mt-2 pb-1 mb-1"/>
                 </div>
-                <div t-field="track.partner_biography" class="oe_no_empty"/>
-                <hr t-if="not is_html_empty(track.description)" class="mt-2 pb-1 mb-1"/>
+            </t>
+
+            <!-- Description -->
+            <div t-if="not is_html_empty(track.description)" t-field="track.description" class="my-2 mx-3 oe_no_empty"/>
```

Comment:
If you only have a "track.description" and nothing else, then you will not have any "hr" tag to split I think, as you will not enter the block here above.

Not sure if intended or not, maybe to double-check.